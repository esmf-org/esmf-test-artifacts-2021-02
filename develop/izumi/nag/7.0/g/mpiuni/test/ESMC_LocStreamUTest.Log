20210227 210943.698 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210227 210943.698 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20210227 210943.698 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20210227 210943.698 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20210227 210943.698 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20210227 210943.698 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20210227 210943.698 INFO             PET0 Running with ESMF Version   : ESMF_8_2_0_beta_snapshot_00-6-gf8f29dec9e
20210227 210943.698 INFO             PET0 ESMF library build date/time: "Feb 27 2021" "19:32:40"
20210227 210943.698 INFO             PET0 ESMF library build location : /project/esmf/himanshu/nag/esmf
20210227 210943.698 INFO             PET0 ESMF_COMM                   : mpiuni
20210227 210943.699 INFO             PET0 ESMF_MOAB                   : enabled
20210227 210943.699 INFO             PET0 ESMF_LAPACK                 : enabled
20210227 210943.699 INFO             PET0 ESMF_NETCDF                 : disabled
20210227 210943.699 INFO             PET0 ESMF_PNETCDF                : disabled
20210227 210943.699 INFO             PET0 ESMF_PIO                    : disabled
20210227 210943.699 INFO             PET0 ESMF_YAMLCPP                : enabled
20210227 210943.699 INFO             PET0 Beginning Test, file ESMC_LocStreamUTest.C, line 53
20210227 210943.699 INFO             PET0 NUMBER_OF_PROCESSORS 1
20210227 210943.700 INFO             PET0 PASS LocStreamCreateLocal, ESMC_LocStreamUTest.C, line 62
20210227 210943.701 INFO             PET0 PASS LocStreamAddKeyAlloc, ESMC_LocStreamUTest.C, line 70
20210227 210943.701 INFO             PET0 PASS LocStreamGetKeyPtr, ESMC_LocStreamUTest.C, line 89
20210227 210943.701 INFO             PET0 PASS LocStreamGetKeyPtr, ESMC_LocStreamUTest.C, line 99
20210227 210943.701 INFO             PET0 PASS LocStreamGetBounds, ESMC_LocStreamUTest.C, line 107
20210227 210943.702 INFO             PET0 PASS Create ESMC_Field object on LocStream, ESMC_LocStreamUTest.C, line 116
20210227 210943.702 INFO             PET0 PASS Destroy ESMC_Field object from LocStream, ESMC_LocStreamUTest.C, line 124
20210227 210943.702 INFO             PET0 PASS Destroy ESMC_LocStream object, ESMC_LocStreamUTest.C, line 132
20210227 210943.702 INFO             PET0 Ending Test, file ESMC_LocStreamUTest.C, line 136
20210227 210943.702 INFO             PET0 Finalizing ESMF
Sat Feb 27 23:11:46 MST 2021
