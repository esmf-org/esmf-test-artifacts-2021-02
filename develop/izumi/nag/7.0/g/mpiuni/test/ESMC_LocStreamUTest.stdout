/project/esmf/himanshu/nag/esmf/src/Infrastructure/stubs/mpiuni/mpirun: line 41: ./ESMC_LocStreamUTest: No such file or directory
Fri Jan 22 09:02:23 MST 2021
